src/utils/web3Utils.js

```diff
@@ -111,19 +128,20 @@ export const getData = async function() {
     getDaiAllowance.bind(this)()
     getDaiBalance.bind(this)()
     getChaiBalance.bind(this)()
+    getChaiTotalSupply.bind(this)()
 }
 
 const secondsInYear = WadDecimal(60 * 60 * 24 * 365)
 
-export const initBrowserWallet = async function() {
+export const initBrowserWallet = async function(prompt) {
     const store = this.props.store
 
     store.set('walletLoading', true)
+    if (!localStorage.getItem('walletKnown') && !prompt) return
```

Comment:
what's going on here?

Reply:
If you visit chai.money a second time we try to connect to your wallet automatically on page load. I didn't want to always try to connect on page load though because that would lead to a metamask popup every time. LocalStorage is just a fancier cookie.